package.json

```diff
@@ -103,7 +108,9 @@
     "build": {
       "dependencies": [
         "compile",
-        "lint"
+        "lint",
+        "build:schema:project",
+        "build:schema:scratch"
```

Comment:
running `yarn install` after checking out this PR reverts some of your psjon scripts, will need to add the devscripts file to override them